### ISSUE TITLE
Range slider question reverting to TXT when placed after certain questions

### ISSUE THREAD

**Issue by katmelen** [IonDen/ion.rangeSlider#748]
I am creating a 17 step form using PAFE range sliders for a few questions. I have one question that is selected as a range slider. When I move the question to any place before my 6th question, which is a radial question, it works properly. But when I place it after that question it reverts to a TXT field. I've deleted the field and re-created it with the same problem. Strangely, I have other questions after that point that are range sliders, and are working properly. Any ideas?

The slider is currently hidden behind an "under construction" page.

**Comment by IonDen** [comment]
Nothing I can do without a demo.
You could try reproduce your situation here: https://jsfiddle.net/IonDen/uqs7njp9/

**Comment by katmelen** [comment]
You can still log into Wordpress with the credentials that I sent. Then you
can see the forms.


Kathleen Melen
(707)239-6632
***@***.***

**Comment by IonDen** [comment]
What?

**Comment by katmelen** [comment]
I can't be the only one using this plug in to set up a new website. Yes,
it's under construction,  which is why you have to log into the website to
see it. That is why I sent login credentials in the original ticket.

Are you suggesting that the only way to troubleshoot a 17 step form is to
recreate that 17 step form on your platform? At this time, I'm getting
quite frustrated with this plug-in.

**Comment by IonDen** [comment]
Which website? Which credentials?

**Comment by katmelen** [comment]
Hi Dennis,

I'm realizing that this must be a different thread than I thought it was. I
did send a message to ***@***.*** on Wednesday. Here's what it said:

Hello,

I am trying to create several long multistep forms using the range slider
for a few of the steps. On both of the forms that I am working on, some of
the range sliders are reverting back to TXT. If I don't change any settings
on the section, but move the questions earlier in the form, they work.
However, this disrupts the flow of the form, placing the questions out of
order. Can you please help me identify what is causing this problem?

The site is currently under construction. So you'll have to log in to see
the forms. Here's the login information:
https://leftcoasthomemortgageteam.com/wp-admin
User Name: Poitnet
Password: s3B5d($yxb^^TRzc4i*&Xyeg

The two forms are in popups attached to the two buttons on the home page.
Here's a listing of the fields that should be range sliders:

"Home Purchase Qualifier"

   - Purchase price - Step 3 - Working correctly
   - Percent Down - Step 8 - Working correctly
   - Income - Step 11 - Reverting to TXT

Additional notes:

   - Zip Code - Step 1 - Set to 5 characters max - Allows any amount of
   characters
   - Cell Phone - Step 17 - Does not force telephone formatting even though
   it's set as phone number

"Refinance Rate Checker"

   - Property value estimate - Step 5 - Showing two range sliders. Can't
   figure out where this is coming from.
   - Remaining mortgage balance - Step 6 - Reverting to TXT
   - Mortgage interest rate - Step 7 -  Showing two range sliders.
   - Additional Cash - Step 11 -  Showing two range sliders. Showing box
   with the numerical amount indicated from the slider. This should not be
   showing and no code has been entered.
   - Income - Step 15 -  Showing two range sliders.  Showing box with the
   numerical amount indicated from the slider.
   - Expenses - Step 16 -  Showing two range sliders.  Showing box with the
   numerical amount indicated from the slider.

Additional notes:

   - Progress bar - cannot get to stay on one line
   - Zip Code - Step 1 - Set to 5 characters max - Allows any amount of
   characters
   - Cell Phone - Step 17 - Does not force telephone formatting even though
   it's set as phone number


Finally, is there a CSS setting somewhere that I can use to edit the color
of the slider to match my website?

Thank you!

Kathleen Melen
(707)239-6632
Kathleen@KathleenMelen.com
Kathleen Melen
(707)239-6632
***@***.***

**Comment by IonDen** [comment]
@katmelen I understand that you are frustrated already, because you can't find the fix for some bug on your website project.

But let me explain you something.
1. I created Ion.RangeSlider library on my free time for free.
2. I released it in Open Source and let every other developer around the world to use it in their projects for free. You could even read its licence: https://github.com/IonDen/ion.rangeSlider/blob/master/License.md
3. That means that I have no relation to your project at all. And I have no time to dig in to it. I have a full time job and family.
4. I'm not paid to fix problems in your project.
5. I can still help you, but all what I ask - prepare a case, show videos, screenshots, code, errors, anything. Or at least direct link to the problem area. I have no idea how your project works and how to reproduce your bug.

**Comment by katmelen** [comment]
I'm sorry Dennis. I understand what you're saying. So this company Poitnet
used your range slider features to develop their PAFE plugin for Elementor?
I purchased the pro version. And that's why I'm frustrated. They also said
they would support it. However, I sent them emails and also, through a link
on some of their documentation, I guess I arrived at your thread on Github.
So if that organization is not associated with you, then it's not you that
I am frustrated with. Unfortunately, at this time, I think I'm going to
have to find another tool.


Kathleen Melen
(707)239-6632
***@***.***
